qbraid_qir/qiskit/visitor.py

```
@@ -0,0 +1,331 @@
+        clabels = [self._clbit_labels.get(bit) for bit in cargs]
+        qubits = [pyqir.qubit(self._module.context, n) for n in qlabels]
+        results = [pyqir.result(self._module.context, n) for n in clabels]
+
+        labels_str = ", ".join([str(label) for label in qlabels + clabels])
+        logger.debug("Visiting instruction '%s' (%s)", instruction.name, labels_str)
+
+        op_name = instruction.name.lower()
+
+        # Handle measurement
+        if op_name in ("measure", "m", "mz"):
+            for qubit, result in zip(qubits, results):
```

Ruff (0.15.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)